### PR TITLE
feat: add keybindings to cycle between keyboard layouts

### PR DESCRIPTION
**Pull request by SoumyaRanjanPatnaik**
Fixes regolith-linux/regolith-desktop/issues/955

### CODE REVIEW

**Comment by SoumyaRanjanPatnaik** [comment]
The following `i3status-rs` block might also be useful for monitoring (and changing) the current keyboard layout.

```
[[block]]
block = "keyboard_layout"
format = "\u2328 $layout ($variant)"
driver = "sway"
interval = 15
[[block.click]]
button = "left"
cmd = "swaymsg input '*' xkb_switch_layout prev"
[[block.click]]
button = "right"
cmd = "swaymsg input '*' xkb_switch_layout next"
```

File: debian/control [hunk 1]
```diff
@@ -387,3 +388,9 @@ Architecture: any
 Depends: ${misc:Depends},
   regolith-wm-config
 Description: Configurations to control lid close behaviour
+
+Package: regolith-sway-kbd-layout
```
Comment (kgilmer) [inline debian/control:392]: nit/opinion: I prefer not abbreviating words in human-exposed strings (unless there is some rt benefit), just for general readability.  But it also is obvious to me that `kbd` is keyboard.

File: usr/share/regolith/sway/config.d/30_keyboard_layout [hunk 2]
```diff
@@ -0,0 +1,15 @@
+###############################################################################
+# Cycle Keyboard layout
+###############################################################################
+
+
+## Modify // Next Layout // <><Alt> BackSpace ##
+set_from_resource $wm.binding.next_layout wm.binding.next_layout Mod1+BackSpace
+set_from_resource $wm.program.next_layout wm.program.next_layout input * xkb_switch_layout next
```
Comment (kgilmer) [inline usr/share/regolith/sway/config.d/30_keyboard_layout:8]: question: what is the `input *` doing?  Is this like some kind of internal command thing that sway interprets?

**Review by kgilmer — APPROVED** [review]

File: usr/share/regolith/sway/config.d/30_keyboard_layout (hunk 2 above)
Comment (SoumyaRanjanPatnaik) [reply usr/share/regolith/sway/config.d/30_keyboard_layout:8]: Layouts in sway are configured on a per device basis, and hence require a selector. The `*` in `input *` selects all the input devices while updating the layout in order to stay consistent with the flashback session.

**Pull request merged**